adafruit_tinylora/adafruit_tinylora.py

```diff
@@ -27,10 +27,31 @@
 
 import time
 from random import randint
-from micropython import const
+
 import adafruit_bus_device.spi_device
+from micropython import const
+
 from adafruit_tinylora.adafruit_tinylora_encryption import AES
 
+try:  # typing
+    from types import TracebackType
+    from typing import Optional, Type, Union
+
+    import busio
+    import digitalio
+    from typing_extensions import Self  # Python <3.11
+    from typing_extensions import Annotated, TypeAlias
+
+    # type aliases
+    bytearray2: TypeAlias = Annotated[bytearray, 2]
+    bytearray4: TypeAlias = Annotated[bytearray, 4]
+    bytearray16: TypeAlias = Annotated[bytearray, 16]
+
+    registeraddress: TypeAlias = Union[const, int]
```

Comment:
registeraddress might be one to consider adding to `circuitpython_typing` we definitely have many libraries with arguments for register addresses. Many of them are typed `int` I believe, but giving them a descripting name that is more specific might be an improvement. @tekktrik you think that one might be worth having over there?